File: C/Sum.c
#include <stdio.h>
int main() {
	int a, b, c, do_not_close_the_window;
	printf(" Введите первое слагаемое:");
	scanf("%d", &a);
	printf("Введите второе слагаемое:");
	scanf("%d", &b);
	c = a + b;
    printf("%d + %d = %d\n", a, b, c);
    scanf("%d", &do_not_close_the_window);
    return 0;
}

File: JavaScript/Sum.js
//let a = prompt('Введите первое слагаемое')
//let b = prompt('Введите второе слагаемое')
//a = Number.parseInt(a)
//b = Number.parseInt(b)
//c= a + b
//alert(a + ' + ' + b + ' = ' + c)

//doNOTcloseTHEwindow = prompt('Не вводи число')


let age
const a = prompt('Сколько Вам лет?')
if (a > 18) {
	age = true
}
else {
	age = false
}
age ? alert('Заходи, бритшка') : alert('Топай в школу')